### ISSUE TITLE
Typo in the strports docstring

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5055]
|<img alt="nkvoll's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| nkvoll reported|
|-|-|
|Trac ID|trac#5055|
|Type|defect|
|Created|2011-04-18 15:02:37Z|

http://twistedmatrix.com/documents/current/api/twisted.application.strports.html#service

Currently it says:

```
    default	Do not use this parameter. It is deprecated since Twisted

    0.
```

Attachments:

* [mypatch.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/29d/29ddae77b611c4223d4f41e229f2d74a6aeba0f9/9b1748c285b9beb56f91af478872472c3811f615.patch) (513 bytes) - added by gdeng on 2012-04-01 21:33:14Z - 
* [strportspatch.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/29d/29ddae77b611c4223d4f41e229f2d74a6aeba0f9/62eeb35db9dadff59c210d0da50f41b62e6d641a.patch) (629 bytes) - added by gdeng on 2012-04-02 20:48:31Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__5055 5055
type__defect defect
reporter__nkvoll nkvoll
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1303138957000000 1303138957000000
changetime__1333819579000000 1333819579000000
version__None None
owner__thijs thijs
cc__thijs cc__jesstess
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_1"></a>|@thijstriemstra commented|
|-|-|

This looks like an issue with pydoctor or something to me..

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)|@jesstess set owner to @jesstess|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_3"></a>|@jesstess commented|
|-|-|

This is an epydoc bug; the "10.2.0." on a line by itself is matching on a regex for if something is a bullet. I've opened https://sourceforge.net/tracker/?func=detail&aid=3503223&group_id=32455&atid=405618 for this issue.

Here's the minimal reproducer from the epydoc ticket:

```

from epydoc.markup import epytext

docstring = """                                                                                                                                                                                                  
    Return the service corresponding to a description.                                                                                                                                                           
                                                                                                                                                                                                                 
    @param default: Do not use this parameter.  It is deprecated since Twisted                                                                                                                                   
        10.2.0.                                                                                                                                                                                                  
"""
print epytext.parse_docstring(docstring, [])
```

which produces

```

<epytext><para>Return the service corresponding to a description.</para><fieldlist><field><tag>param</tag><arg>default</arg><para inline=True>Do not use this parameter.  It is deprecated since Twisted</para><olist start='0'><li bullet='10.2.0.'></li></olist></field></fieldlist></epytext>
```

epydoc doesn't seem to be maintained anymore, so if I don't hear back on the ticket soon the easiest thing to do here is to tweak the line a bit to not match the buggy regex.

**Comment by twisted-trac** [comment]
|<img alt="gdeng's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|gdeng set owner to gdeng|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="gdeng's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|gdeng removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)<a name="note_6"></a>|@itamarst set owner to gdeng|
|-|-|

Thanks for the patch! There are a couple of minor problems:

  1. While that does look like it would solve the problem, it also violates our coding standard line length limit (http://twistedmatrix.com/documents/current/core/development/policy/coding-standard.html#auto5); maybe keep the lines wrapped, but rephrase the sentence somehow so "10.2.0" isn't the last thing on the line?
  1. Patches should include news files, as documented here - http://twistedmatrix.com/trac/wiki/ReviewProcess#Newsfiles - in this case you'd want an empty .misc file since it's a trivial change. If you `svn add` the news file it will show up in `svn diff` output.

Please add a new patch and re-add the review keyword and we can get this merged.

**Comment by twisted-trac** [comment]
|<img alt="gdeng's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|gdeng removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_8"></a>|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

Thanks I will apply the patch.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_9"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [34127]) Apply strportspatch.patch: Correct formatting in the docstring of t.application.strports.service.

Author: gdeng
Reviewer: jesstess, itamar, thijs
Fixes: [#5055](https://github.com/twisted/twisted/issues/5055)
